features/facebookImport/src/components/scrollableXGradient/scrollableXGradient.css

```diff
@@ -0,0 +1,33 @@
+.startScroll .scrollable {
+    position: relative;
+    overflow: hidden;
+    display: flex;
+    flex-direction: row;
+}
+
+.startScroll h1 {
+    font-size: 20px;
+    margin: 4px 0px !important;
+    white-space: nowrap;
+    overflow: scroll hidden;
+}
+/* Hide scrollbar for Chrome, Safari and Opera */
+.startScroll h1::-webkit-scrollbar {
+    display: none;
+}
+
+/* Hide scrollbar for IE, Edge and Firefox */
+.startScroll h1 {
```

Comment:
Probably it's just my ignorance, but can't you put this together with the same group above? Just add these two lines behind line 12?

Reply:
It could, but I think it's separate because it's a dedicated workaround, I find it pretty readable this way.